### DISCUSSION TITLE
Converts with low SV not converting correctly in taiko and mania

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#17120] [Q&A] [answered]
Originally mentioned in #15747 and #15808
> https://osu.ppy.sh/b/1494828 At 02:32:949, there's a 0.01x velocity point with a slider that lasts until 02:35:949. In taiko and mania, this note lasts until approximately 03:03. 
> ![osu_2021-11-22_12-16-07](https://user-images.githubusercontent.com/74262407/142932680-dca9dfdf-d656-4802-a77c-e532bce2fabd.png)
> ![osu_2021-11-22_12-35-31](https://user-images.githubusercontent.com/74262407/142932742-1bd6b8d2-a470-4ed9-8536-9fa6aabfa03b.png)
> ![osu_2021-11-22_12-35-55](https://user-images.githubusercontent.com/74262407/142932757-1762c0be-c56e-4986-b143-10035bb3a747.png)

While originally discarded as a duplicate of #14835, I don't think that's the case. There will inevitably be maps with lower scroll speeds than the allowed limit, and lazer isn't calculating the duration properly for these modes. If std and catch aren't affected, why should taiko and mania?
For clarification, this is NOT about allowing lower scroll speeds. This is about ensuring duration is accurate to allow converts to be played. You can still set the SV to 1 but have the correct duration and have the map be playable. In its current state, the map isn't.

**Comment by bdach** [comment, marked as answer]
this is a duplicate of the aforementioned discussions. i'm not sure why this is being brought up again rather than bumping the old discussion (if there is a need for it, which there isn't).

**Reply by zachmanthethird** [reply]
By increasing the limit, you're not solving the problem. Taiko and mania are calculating durations of sliders and spinners using incorrect values if they are converted from osu!standard maps. Notice how this problem isn't noticeable in taiko-only maps such as https://osu.ppy.sh/b/2862238.

**Reply by bdach** [reply]
...i'm 99% sure increasing the limit will cause conversion to work correctly because the values are getting mangled on read because of lazer's more stringent limits for the difficulty point multiplier.

please. i'd like to think i know what i'm talking about.

**Reply by smoogipoo** [reply]
Tending to think this is different from the aforementioned discussion because this is a breaking change that affects ranked maps and also diffcalc. (https://github.com/ppy/osu/issues/17143)
